### DISCUSSION TITLE
[WS-1300] Small viewport doesn't show tabs in a project's settings

### DISCUSSION THREAD

**Discussion by ngchrbn** [vercel/community#6139] [Bug Report]
### Summary

When I open the project's settings tab and I am on a viewport less than 961px, I can't change the project's settings tabs for example to go Environment variables, General, Domains, ...

### Steps to Reproduce

1. Open a project using a viewport less than 961px
2. Go to the settings tab
3. Try to go to General, Domains, Environment Variables, ... for example

> I am using Brave 1.63.169 Chromium: 122.0.6261.111 (Official Build) (64-bit) and I tested on Google Chrome version 122.0.6261.129 (Official Build) (64-bit)

> Nothing happens if you click on the button with a blue circle from the screenshot below: "< Project settings".

Here is a screenshot.
![image](https://github.com/vercel/community/assets/62961025/0f2e19fa-a084-48a1-bf72-c060d6ffe144)

**Comment by swarnava** [comment]
Thanks for reporting this. I have forwarded this to relevant team.
